mappings/net/minecraft/world/event/listener/DynamicGameEventListener.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/unmapped/C_wpaexsvv net/minecraft/world/event/listener/DynamicGameEventListener
```

Comment:
Also mapped in #141

Reply:
mapped it so it would make a little more sense to mean within `WardenEntity` I shall remove it when marked ready